tests/tool/stylelint_tool_plugin/valid_package/test.html

```diff
@@ -7,12 +7,11 @@
     <style>
       body
       {
-        width: 100%;
       }
     </style>
   </head>
   <body>
     <!-- page content -->
```

Comment:
```suggestion
    <!-- page content with intentional extra space to trigger a stylelint warning -->
```

Reply:
this change is actually just fixing an issue that we intentially added before since that rule has been deprecated.